acpreprocessing/stitching_modules/convert_to_n5/tiff_to_ngff.py

```diff
@@ -573,6 +603,9 @@ def write_mimgfns_to_n5(
                 attributes = group_attributes[i]
             except IndexError:
                 continue
+            if deskew:
+                if "pixelResolution" in attributes:
+                    attributes["pixelResolution"]["dimensions"][0]/=deskew_options["stride"] 
```

Comment:
Not sure we should be modifying the attributes in place -- might be preferable to have a flag related to dimensionality changes when generating the group attributes in the first place.

Reply:
Yes, I think this could be related to the next comment about schematizing the omezarr attributes in improved handling of the attributes in general.